static/app/components/platformPicker.tsx

```diff
@@ -37,6 +37,8 @@ type Props = {
   listProps?: React.ComponentProps<typeof PlatformList>;
   noAutoFilter?: boolean;
   defaultCategory?: Category;
+  organization: Organization | null | undefined;
```

Comment:
@roggenkemper We should keep this as `organization?: Organization`. No need to accept null.